src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -2519,11 +2519,18 @@ class run_impl
   [[nodiscard]] ert_cmd_state
   wait(const std::chrono::milliseconds& timeout_ms) const
   {
+    // dump dtrace buffer if ini option is enabled
+    static auto dtrace_lib_path = xrt_core::config::get_dtrace_lib_path();
```

Comment:
Please make it a bool. 
```suggestion
    static bool dtrace = !xrt_core::config::get_dtrace_lib_path().empty();
```

Reply:
sure. will change